src/state/reducers/auth/index.js

```diff
@@ -44,93 +44,96 @@ export const authReducer = createReducer(
   {
```

Comment:
`createReducer` is still using `redux-act`, we should probably remove that package from the project.